fs/indexed-pkg-importer/src/index.ts

```diff
@@ -1,9 +1,11 @@
-import { constants, type Stats, existsSync } from 'fs'
+import { type Stats, existsSync } from 'fs'
+import { constants } from 'fs/promises'
```

Comment:
why was this change needed?

Reply:
Oh, in the 1st commit you removed `constants` and I added it too, I'll import it from `fs` so it's one line only